Decred Wallet/Features/Settings/ServerSetTableViewController.swift

```diff
@@ -34,14 +34,12 @@ class ServerSetTableViewController: UITableViewController {
     @objc func save() -> Void {
         // save here
         if (server_ip.text?.isEmpty)! || (server_ip.text)! == ""{
```

Comment:
I think this if statement can be simplified with a guard statement
`guard let ipAddress = server_ip.text, ipAddress != "" else {`